packages/split-view/src/spectrum-split-view.css

```diff
@@ -0,0 +1,364 @@
+/* stylelint-disable */ /* 
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+
+THIS FILE IS MACHINE GENERATED. DO NOT EDIT */
+:host {
+    /* .spectrum-SplitView */
+    --spectrum-splitview-vertical-width: 100%;
+    --spectrum-splitview-vertical-gripper-width: 50%;
+    --spectrum-splitview-vertical-gripper-outer-width: 100%;
+    --spectrum-splitview-vertical-gripper-reset: 0;
+    display: flex;
+    overflow: hidden;
+}
+::slotted(*) {
+    /* .spectrum-SplitView-pane */
+    height: 100%;
+}
+:host([dir='ltr']) #gripper {
```

Comment:
I feel like this matches https://github.com/adobe/spectrum-web-components/pull/1154/files#diff-21070cfdf1a047d11b4140e4f2838f99b136ad0fcfa5da0efa8b1bf6b2b43652R109, can you confirm that we need the custom styles, too.

Reply:
It feels like we may be doing this in a couple of places in the CSS of `split-view.css`. To clarify, could we maybe add some comments to the customization to clarify why they are there?

Reply:
All of the measurements that we persist in `split-view.css` may also need updated CSS Custom Property names.

Reply:
We need both matchers, as we have a custom splitter size (with margin+padding to allow hover states) and the difference is the first offset: `var(--spectrum-global-dimension-static-size-125) +` - I can add some comment in the custom css.

Reply:
I tried out the translateX/Y css function, but that needed to much restructuring, so I re-formatted and added further comments about the additional added offset.